tools/repo/notify.py

```diff
@@ -278,6 +279,13 @@ def pr_message(self, age, pull):
             f"<{pull['html_url']}|{html.escape(pull['title'])}> has been waiting "
             f"{markup}{days} days {hours} hours{markup}")
 
+    async def track_open_issues(self):
+        response = await self.session.get(
+            "https://api.github.com/repos/envoyproxy/envoy/issues?labels=triage")
+        content = str(await response.read(), 'UTF-8')
+        parsed = await response.json()
+        return len(parsed)
```

Comment:
```suggestion
        return len(await response.json())
```

i think this is all thats needed